cmd/manager/main.go

```diff
@@ -63,6 +64,8 @@ const (
 	CertValidityFlag       = "cert-validity"
 	CertRotateBeforeFlag   = "cert-rotate-before"
 
+	CheckReferencesRbac = "check-references-rbac"
```

Comment:
I know this has already been renamed once and I don't want to bikeshed too much but what about `check-rbac-on-refs` or `enforce-rbac-on-refs` somthing that makes it clear that we are not checking the references but RBAC rules when working with references

Reply:
+1 that's better